cmake/modules/LXQtCompilerSettings.cmake

```diff
@@ -122,6 +129,20 @@ endif()
 
 
 #-----------------------------------------------------------------------------
+# Turn on more aggrassive optimizations not supported by CMake
+# References: https://wiki.qt.io/Performance_Tip_Startup_Time
+#-----------------------------------------------------------------------------
+if (CMAKE_COMPILER_IS_GNUCXX OR LXQT_COMPILER_IS_CLANGCXX)
+    # -flto: use link-time optimizations to generate more efficient code
+    # -Bsymbolic-functions: replace dynamic symbols used internally in 
+    #                       shared libs with direct addresses.
+    set(LTO_FLAGS "-flto -fuse-linker-plugin")
```

Comment:
Do you intend to use the `LDO_FLAGS` elsewhere? Or why are these options set into separate variable and the other not?